<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Emotion</class>
 <widget class="QWidget" name="Emotion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>155</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>155</width>
     <height>129</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion01.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>0</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion02.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>0</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion03.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>0</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion04.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>0</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion05.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion06.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>26</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion07.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>26</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion08.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>26</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion09.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>26</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion10.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>26</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion11.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>26</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion12.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>52</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion13.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>52</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion14.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>52</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion15.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>52</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion16.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>52</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion17.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>52</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion18.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>78</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion19.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>78</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion20.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>78</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion21.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>78</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion22.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>78</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion23.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>78</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion24.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>104</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion25.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>104</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion26.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>104</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion27.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>104</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion28.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>104</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion29.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>104</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(':/emotion/emotion/emotion30.png');
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>label_15</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
